### ISSUE TITLE
SuperSlicer build fail.

### ISSUE THREAD

**Issue by nanoplane** [supermerill/SuperSlicer#974]
### Version
2.3.55.5
### Operating system type + version
Mac OS Big Sur 11.2.1

### 3D printer brand / version + firmware version (if known)
N/A
### Behavior
Successfully ran cmake and make for deps.

Building SuperSlicer in xCode:
Can't complete BUILD_ALL due to multiple failures.

Build of SuperSlicer fails due to missing deps library: libwx_osx_cocoau_scintilla-3.1.a
Since deps build succeeded, I'm not sure where it's dropping the lib....

**Comment by royal32** [comment]
I got past this by copying `deps/build/destdir/usr/local/lib/libwxscintilla-3.1.a` as `libwx_osx_cocoau_scintilla-3.1.a`. So the same file exists with both names. 
I'm not familiar enough with compilation to determine the root cause, though.

**Comment by nanoplane** [comment]
So, did that and build succeeds but running it crashes on launch in  wxMacCoreGraphicsPenBrushDataBase::CalculateShadingValues
... bummer..

**Comment by royal32** [comment]
Yup... tracking that issue [here](https://github.com/supermerill/SuperSlicer/issues/777), also mentioned [here.](https://github.com/supermerill/SuperSlicer/issues/936).

**Comment by nanoplane** [comment]
So, looks like it's been solved. There's a comment that changes wxWidgets.cmake to use the wxWidgets 3.1.4-patched version for all Apple, not just arm..

**Comment by royal32** [comment]
Yup, figured it out about an hour ago. I'd expect fixed builds to come pretty soon

**Comment by ctilley83** [comment]
So it's building correctly, but the beta mac binaries posted a couple days ago simply crash on start? I'm experiencing the same thing on Big Sur 11.2.1

**Comment by nanoplane** [comment]
Yep, crashes for me as well.
